libraries/rush-lib/src/logic/operations/OperationExecutionRecord.ts

```diff
@@ -196,7 +196,8 @@ export class OperationExecutionRecord implements IOperationRunnerContext {
         ? new ProjectLogWritable(
             associatedProject,
             this.collatedWriter.terminal,
-            `${associatedPhase.logFilenameIdentifier}${logFileSuffix}`
+            `${associatedPhase.logFilenameIdentifier}${logFileSuffix}`,
+            true
           )
```

Comment:
With this many arguments, consider making this an `options` object with named properties.

Reply:
Updated in [e531b9c](https://github.com/microsoft/rushstack/pull/4668/commits/e531b9cde8d88c03e0499d6f4a4ab714b9a8bf95)